src/backend/web/templates/event_list.html

```diff
@@ -31,7 +31,7 @@
           </ul>
         </div>
 
-        <h1 class="visible-xs end_header">{% if explicit_year %}{{selected_year}} {% endif %}<em>FIRST</em> Robotics Competition Events{% if state_prov %} in {{state_prov}}{% endif %}{% if events|length > 0 %} <small>{{ events|length }} Events</small>{% endif %}</h1>
+        <h1 class="visible-xs end_header">{% if explicit_year %}{{selected_year}} {% endif %}<em>FIRST</em> Robotics Competition Events{% if state_prov %} in {{state_prov}}{% endif %}{% if events|length > == 1 %} <small>{{ events|length }} Event</small>{% endif %}{% if events|length > 1 %} <small>{{ events|length }} Events</small>{% endif %}</h1>
```

Comment:
There's

```
{% if events|length > == 1 %}
```

And the `> ==` is what's failing to parse

Reply:
Bad copy pasta!